### PR TITLE
DG: Add effort

### PR DESCRIPTION
**Pull request by alientian**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1078,3 +1130,56 @@ Prerequisite: There is at least one company in the list.
 <div style="page-break-after: always;"></div>
 
 ## **Appendix D: Effort**
+### UI Enhancements
+The UI has been revamped to allow card-based viewing of recruiter details.
+The main panel has been split into the company detail panel and the company list panel.
```
Comment (GlendaChong) [inline docs/DeveloperGuide.md:1135]:
```suggestion
The main panel has been split into the `CompanyDetailPanel` and the `CompanyListPanel`.
```

**Review by tayruxin — APPROVED** [review]
LGTM

**Review by GlendaChong — CHANGES_REQUESTED** [review]

**Review by GlendaChong — APPROVED** [review]

File: docs/UserGuide.md [hunk 2]
```diff
@@ -442,7 +445,7 @@ Add the company to LinkMeIn now to keep track of the application process using t
 **Format**:
 `add c/COMPANY_NAME r/ROLE s/APPLICATION_STATUS d/DEADLINE n/RECRUITER_NAME e/EMAIL p/PHONE_NUMBER [pr/PRIORITY]`
 
-> All [**parameters**](#parameters-description) specified in the format are compulsory except for PRIORITY.
+> All [**parameters**](#parameters-description-brbr) specified in the format are compulsory except for PRIORITY.
```
Comment (tayruxin) [inline docs/UserGuide.md:448]: pls check

File: docs/UserGuide.md [hunk 3]
```diff
@@ -548,10 +551,12 @@ _After adding a company, you wish to note down more information such as the job
...
 </div>
 
-**Examples:** 
+**Examples:**
 
 **Step 1.** After adding the internship for DSTA into LinkMeIn,
-   you want to note that this internship requires experience in Java.
+you want to note that this internship requires experience in Java.
+
+<div style="page-break-after: always;"></div><br/>
```
Comment (tayruxin) [inline docs/UserGuide.md:559]: check

File: docs/UserGuide.md [hunk 4]
```diff
@@ -874,7 +879,7 @@ Encountered an unfamiliar term when using LinkMeIn? Read the respective descript
 
 <div style="page-break-after: always;"></div><br/>
 
-### Parameters Description
+### Parameters Description <br/><br/>
```
Comment (tayruxin) [inline docs/UserGuide.md:882]: check

**Review by tayruxin — CHANGES_REQUESTED** [review]
All those need to look into

File: docs/UserGuide.md [hunk 5]
```diff
@@ -25,18 +25,18 @@ Moreover, LinkMeIn combines the **efficiency** of the [Command Line Interface (C
 
 # Table of Contents
 * TOC
-{:toc}
+  {:toc}
```
Comment (tayruxin) [inline docs/UserGuide.md:28]: will this affect the TOC?

Comment (alientian) [reply docs/UserGuide.md:28]: I will just edit it

**Review by tayruxin — APPROVED** [review]

**Pull request merged**